Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/common/azure/serverless/AzureSparkServerlessCluster.java

```diff
@@ -57,6 +59,7 @@
                                                     DestroyableCluster,
                                                     LivyCluster,
                                                     YarnCluster,
+                                                    IClusterDetail,
```

Comment:
>IClusterDetail [](start = 52, length = 14)

Notice that `SparkCluster` inherits from `IClusterDetail`, so we can remove the `IClusterDetail` here.